x-pack/plugins/index_management/__jest__/client_integration/helpers/http_requests.ts

```diff
@@ -107,6 +107,39 @@ const registerHttpRequestMockHelpers = (server: SinonFakeServer) => {
     ]);
   };
 
+  const setLoadIndexMappingResponse = (response?: HttpResponse, error?: ResponseError) => {
+    const status = error ? error.statusCode || 400 : 200;
+    const body = error ?? response;
+
+    server.respondWith('GET', `${API_BASE_PATH}/mapping/:name`, [
+      status,
+      { 'Content-Type': 'application/json' },
```

Comment:
would it make sense to represent this content type block as a constant HEADERS or something to avoid repeating or is there an expectation these might be different across tests?

Reply:
yes, this file is very repetitive. I'll see how all the same lines can be refactored into a function or similar.